framework/src/geomsearch/NearestNodeLocator.C

```diff
@@ -85,7 +87,7 @@ NearestNodeLocator::findNodes()
     std::vector<dof_id_type> trial_master_nodes;
 
     // Build a bounding box.  No reason to consider nodes outside of our inflated BB
-    BoundingBox * my_inflated_box = NULL;
+    UniquePtr<BoundingBox> my_inflated_box = NULL;
```

Comment:
`std::unique_ptr<BoundingBox> my_inflated_box`; `UniquePtr` is a libMesh shim class that will go away. You can use the C++11 object directly. You don't need to initialize it either, but since you did. Don't use `NULL` anymore. It's just a typedef to 0 which can get you in trouble. In C++11 you should use `nullptr` which is an actual strongly typed null value. I know this is nit-picking but I'd like to train developers to start writing more modern code.

Reply:
Thanks for the comment @permcody. I am still not very familiar with the C++11 syntax/capabilities. These comments will help me learn C++11.